eclipse-collections/src/main/java/org/eclipse/collections/impl/list/Interval.java

```diff
@@ -176,6 +176,26 @@ public static Interval fromTo(int from, int to)
         return Interval.fromToBy(from, to, -1);
     }
 
+    /**
+     * Returns an Interval starting from the value from until the specified value to (exclusive) with a step value of 1
+     */
+    public static Interval fromUntilTo(int from, int to)
+    {
+        if (to == Integer.MIN_VALUE)
+        {
+            throw new IllegalArgumentException("to cannot be the Integer minimum value " + Integer.MIN_VALUE);
+        }
+
+        int until = to - 1;
```

Comment:
This will not work for negatives. For negatives it needs to be `to+1`

Reply:
I think this should be `to - 1` even for negatives.

// For example this is what will happen in the Kotlin's `until`
```kt
for (i in -10 until -1) {   
    print(i) // -10-9-8-7-6-5-4-3-2
}
```

Reply:
Hi @sendilkumarn , I think @nikhilnanivadekar comment above is why `Interval.fromToExclusive(0, -9)` is returning a size of 11.

Reply:
This is tricky especially because we support both forward and reverse interval generation in the same API. 

How about this,

`
if (to < 0) {
    if (from <  to) {
        to = to - 1;
    } else {
       to = to + 1;
    }
} 
`

```
Interval.fromToExclusive(0, -9)  => 0, -1, -2, -3, -4, -5, -6, -7, -8 

Interval.fromToExclusive(-10, -5)  => -10, -9, -8, -7, -6

```

What do you think?

Reply:
Looks good. Tests should validate.

Reply:
What about the following?

`Interval.fromToExclusive(10, 0)`

Reply:
Interval.fromToExclusive(10, 0) // 10, 9, 8, 7, 6, 5, 4, 3, 2, 1
Interval.fromToExclusive(100, 50)  // 100, 99, ..... 51